api/SVGTransformList.json

```diff
@@ -57,7 +57,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "79"
```

Comment:
This doesn't seem like it can be right. A `FooList` interface without a `length` attribute would be very unusual. http://mdn-bcd-collector.appspot.com/tests/api/SVGTransformList tests the prototype only, so I think that's the problem here.